app/grants/utils.py

```diff
@@ -290,3 +294,11 @@ def sync_payout(contribution):
 
     if subscription.tenant == 'ZCASH':
```

Comment:
check this out:
```
tenant_payout_mapper = {
    'CELO': sync_celo_payout,
    'ZIL': sync_zil_payout,
    'POLKADOT': sync_polkadot_payout,
    'HARMONY': sync_harmony_payout}

tenant_payout_mapper[subscription.tenant](contribution)
```

put the tenant payout_mapper as a "global" in this submodule so it is loaded into memory when the server boots for even more performance.